### PR TITLE
Bug 1147076 - Ensure drawn detected face circles are centred on the same point as camera hardware.

### PR DESCRIPTION
**Pull request by aosmond**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=275828258d1089bfdc2fd904059906b553e004d1

File: apps/camera/js/controllers/viewfinder.js [hunk 1]
```diff
@@ -237,10 +237,12 @@ ViewfinderController.prototype.onFacesDetected = function(faces) {
 };
 
 ViewfinderController.prototype.calculateFaceCircle = function(face) {
+  var diameter = Math.max(face.width, face.height);
+  var radius = diameter/2;
```
Comment (mikeaich) [inline apps/camera/js/controllers/viewfinder.js]: diameter<space>/<space>2

File: apps/camera/js/controllers/viewfinder.js [hunk 2]
```diff
@@ -237,10 +237,12 @@ ViewfinderController.prototype.onFacesDetected = function(faces) {
...
   return {
-    x: face.left,
-    y: face.top,
-    diameter: Math.max(face.width, face.height)
+    x: Math.round(face.left + face.width/2 - radius),
+    y: Math.round(face.top + face.height/2 - radius),
```
Comment (mikeaich) [inline apps/camera/js/controllers/viewfinder.js]: <space>/<space>2

File: apps/camera/js/controllers/viewfinder.js (hunk 1 above)
Comment (mikeaich) [inline apps/camera/js/controllers/viewfinder.js]: So this was supposed to be &lt; space &gt; / &lt; space &gt; 2 ... but github thinks it's smarter than I am.

File: apps/camera/js/controllers/viewfinder.js (hunk 2 above)
Comment (mikeaich) [inline apps/camera/js/controllers/viewfinder.js]: Ditto.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3261c9724bdfc2d30c3db21fb364145ea9f58b07

**Pull request merged**